modules/ROOT/pages/cassandra/cassandra-design-center.adoc

```diff
@@ -0,0 +1,118 @@
+= Design Center - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configure the Cassandra Connector in Design Center
+. From Anypoint Platform > Design Center, create a Mule Application, and add an HTTP Listener as the trigger.
++
+. Create an HTTP global element for the connector, by setting these fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Protocol | Protocol selected for the HTTP endpoint, it can be HTTP or HTTPS (secure).
+|Host| IP address where the Mule application listens for requests.
+|Port| Port address where the Mule application listens for requests.
+|Base Path| Path where the Mule application listens for requests.
+|===
++
+. Select the plus sign to add a component.
+. Select the connector as a component.
+. Select the operation.
++
+[%header%autowidth.spread]
+|===
+|Operation |Description
+|Add New Column a|Input for this operation is:
+
+* Parameter: A table name
+* Optional keyspace name
+* Payload: an AlterColumnInput POJO
++
+The POJO has the following fields:
++
+** Name of the column to be created
+** The data type for the new column
+
+|Change Column Type a|  Input for this operation is:
+
+* Parameter: A table name
+* Optional keyspace name
+* Payload: an AlterColumnInput POJO
+|Create Keyspace a| Input for this operation using SimpleStrategy is:
+
+* Keyspace name
+* Replication factor
+* Replication Strategy Class
+
+|Create Table a| Input for this operation is:
+
+* Column names
+* Table names
+* Optional keyspace name
+
+|Delete Columns Value a| Input for this operation is:
+
+** Parameter: A table name as a parameter
+** Optional keyspace name
+** Payload: a `Map<String, Object>` having two records with the keys `where` and `columns`
+|Delete Rows a| Input for this operation is:
+
+* Parameter: A table name
+* Optional keyspace name
+* Payload a `Map<String, Object>` with one record having the key `where` and a value `Map<String, Object>` containing the WHERE clause.
+|Drop Keyspace | This operation accepts a String parameter representing the keyspace name.
+|Drop Table a| Input for this operation is:
+
+* A table name
+* The keyspace that contains the table
+|Execute CQL Query  a| Transform Message Payload.
```

Comment:
I don't think payload should be initial capped,

Reply:
Will need to double check on this one.